LaunchAtLogin/copy-helper.sh

```diff
@@ -18,5 +26,5 @@ fi
 
 if [[ $CONFIGURATION == "Release" ]]; then
 	rm -rf "$origin_helper_path"
-	rm "$(dirname "$origin_helper_path")/copy-helper.sh"
+	rm "$(dirname "$origin_helper_path")/copy-helper.sh" || true
```

Comment:
Why are you silencing errors?

Reply:
If there's no `copy-helper.sh` file in that location, that command will return a non-zero value and - bc it's the last command - the script will return that value. When the script returns a non-zero value, it causes that build step to fail.

Reply:
If there's no `copy-helper.sh` file in that location, the `rm` command will return a non-zero value and - bc it's the last command - the script will return that value. When the script returns a non-zero value, it causes that build step to fail.